python/ray/data/tests/test_backpressure_policies.py

```diff
@@ -129,36 +129,6 @@ def test_e2e_normal(self):
         start2, end2 = ray.get(actor.get_start_and_end_time_for_op.remote(2))
         assert start1 < start2 < end1 < end2, (start1, start2, end1, end2)
 
-    def test_e2e_time_backpressure(self):
-        """A simple E2E test with ConcurrencyCapBackpressurePolicy enabled."""
-        # TODO: merge this test with the above once we are exporting the
-        # backpressure_time in op_runtime_metrics.py. This test currently
-        # mutates the OpRuntimeMetrics dataclass to observe the backpressure time.
-        from ray.data._internal.execution.interfaces.op_runtime_metrics import (
-            OpRuntimeMetrics,
-        )
-
-        with patch.object(
-            OpRuntimeMetrics.__dataclass_fields__["task_submission_backpressure_time"],
-            "metadata",
-            {"export": True},
-        ):
-            actor = self._create_record_time_actor()
-            map_func1 = self._get_map_func(actor, 1)
-            map_func2 = self._get_map_func(actor, 2)
-
-            # Create a dataset with 2 map ops. Each map op has N tasks, where N is
-            # the number of cluster CPUs.
-            N = self.__class__._cluster_cpus
-            ds = ray.data.range(N, override_num_blocks=N)
-            # Use different `num_cpus` to make sure they don't fuse.
-            ds = ds.map_batches(map_func1, batch_size=None, num_cpus=1, concurrency=1)
-            ds = ds.map_batches(map_func2, batch_size=None, num_cpus=1.1, concurrency=1)
-            ds.take_all()
-            assert (
-                0 < ds._plan.stats().extra_metrics["task_submission_backpressure_time"]
```

Comment:
let's keep the existing test, but remove the back pressure time check? the existing test without the back pressure time check is still stable right?

Reply:
The test I deleted is basically just a copy of the above `test_e2e_normal` with some changed asserts so deleting the test isn't reducing coverage. The intention was to eventually remove it once we were exporting the backpressure time in `OpRuntimeMetrics`.